src/timeline/InputBar.cpp

```diff
@@ -249,6 +250,54 @@ InputBar::updateAtRoom(const QString &t)
         this->containsAtRoom_ = roomMention;
         emit containsAtRoomChanged();
     }
+
+    // check for invalid commands
+    auto commandName = getCommandAndArgs().first;
+    bool hasInvalidCommand{};
+    if (!commandName.isNull() && '/' + commandName != text()) {
```

Comment:
What is the second condition for?

Reply:
My reasoning is that if you start typing `/roomni`, you don't want it to yell at you until you have finished typing the command. But I'm happy to change that if you think otherwise.

Reply:
Well, with your other changes, that means if I type /clear, that will be sent into the room. I don't know what is the right approach there :)

Reply:
Well, that is definitely not good. I'll fix that. But the underlying question of how to handle invalid commands with no arguments remains.

Reply:
Wait. I just realized that I read that wrong. I agree that I don't know what the correct approach is.